national-institute-of-health-research.csl

```diff
@@ -0,0 +1,204 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" page-range-format="minimal" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>National Institute of Health Research</title>
+    <title-short>NIHR</title-short>
+    <id>http://www.zotero.org/styles/national-institute-of-health-research</id>
+    <link href="http://www.zotero.org/styles/national-institute-of-health-research" rel="self"/>
+    <link href="http://www.zotero.org/styles/international-journal-of-infectious-diseases" rel="template"/>
+    <link href="http://www.journalslibrary.nihr.ac.uk/information-for-authors/references" rel="documentation"/>
+    <author>
+      <name>Rhiannon Miller</name>
+      <email>rhiannon.miller@prepress-projects.eu</email>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="medicine"/>
+    <summary>A style for journals in the NIHR Journals Library, resembles Vancouver style</summary>
+    <updated>2016-02-05T11:44:17+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="access">
+    <choose>
+      <if variable="DOI">
+        <text variable="DOI" prefix="http://dx.doi.org/"/>
+      </if>
+      <else-if type="webpage post-weblog" match="any">
+        <choose>
+          <if variable="URL">
+            <group delimiter=" ">
+              <text variable="URL" prefix="URL: "/>
+              <date variable="accessed" prefix="(Accessed " form="text"/>
```

Comment:
Dear Rintze

Thanks for getting back to me.

The closing ) was on line 185:

        <else-if type="webpage">
          <group delimiter=" ">
            <text variable="title" form="long" font-style="italic" suffix="."/>
            <text variable="container-title" form="long" suffix="." text-case="title"/>
            <text macro="publisher"/>
            <text macro="access" suffix=")"/>
          </group>
        </else-if>	

However, I’ve taken your advice and replaced the code as you suggested. I’ve also made a few other minor changes including changing the default-locale.

Please let me know if there is anything more I need to do at this stage.

Thanks and best wishes

Rhiannon

Prepress Projects
Rhiannon Miller • Programmes Manager
+44 (0)1738 450489

> On 17 Feb 2016, at 16:08, Rintze M. Zelle <notifications@github.com> wrote:
> 
> In national-institute-of-health-research.csl <https://github.com/citation-style-language/styles/pull/1889#discussion_r53186520>:
> 
> > +    <category field="medicine"/>
> > +    <summary>A style for journals in the NIHR Journals Library, resembles Vancouver style</summary>
> > +    <updated>2016-02-05T11:44:17+00:00</updated>
> > +    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
> > +  </info>
> > +  <macro name="access">
> > +    <choose>
> > +      <if variable="DOI">
> > +        <text variable="DOI" prefix="http://dx.doi.org/"/>
> > +      </if>
> > +      <else-if type="webpage post-weblog" match="any">
> > +        <choose>
> > +          <if variable="URL">
> > +            <group delimiter=" ">
> > +              <text variable="URL" prefix="URL: "/>
> > +              <date variable="accessed" prefix="(Accessed " form="text"/>
> Where is the closing ")"?
> 
> (also, in general, it would be better to replace
> 
> <date variable="accessed" prefix="(Accessed " form="text"/>
> with
> 
> <group prefix="(" suffix=")" delimiter=" ">
>   <text term="accessed" text-case="capitalize-first"/>
>   <date variable="accessed" form="text"/>
> </group>
> That way you use a translatable term, which makes it easier to reuse this style in other languages.
> )
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/citation-style-language/styles/pull/1889/files#r53186520>.
>